pxc/local/build-binary-galera3

```diff
@@ -55,6 +55,10 @@ fi
 # ------------------------------------------------------------------------------
 
 pushd ${SOURCEDIR}
-    scons
+    if [[ "${CMAKE_BUILD_TYPE}" = "Debug" ]]; then
+       scons debug=0
+    else
+       scons
```

Comment:
Shouldn't we also specify psi=1 for both cases?

Reply:
psi = 1 should be used only for libgalera_smm.so and not for garbd. garbd crashes if we built with psi=1.(https://jira.percona.com/browse/PXC-3428).

I think we should build libgalera_smm.so and garbd separetely with different arguments.

Reply:
Galera3 build script is only used for PXC-5.6 MTR job and psi=1 is not specified in PXC-5.6 build script. Please let me know if we need to add psi=1 for PXC-5.6 MTR job. 
https://github.com/percona/percona-xtradb-cluster/blob/5.6/build-ps/build-binary.sh#L338